source/server/session_resource_repository.h

```diff
@@ -29,6 +29,8 @@ class SessionResourceRepository {
       InitFunc init_func,
       CleanupSessionFunc cleanup_func,
       uint32_t& session_id,
+      SessionInitializationBehavior initialization_behavior = SESSION_INITIALIZATION_BEHAVIOR_UNSPECIFIED,
```

Comment:
Yes, but the only caller that passed in `include_in_reverse_lookup` was in `add_dependent_session` in this file. For that reason, I updated that call and changed the ordering to keep the `include_in_reverse_lookup` arg at the end so that all the generated calls don't have to specify it. They can just pass in the two new arguments and leave off that one to get the default `true` for it.

Reply:
Also, this isn't code exposed to external clients either - it's just internal to the server implementation.